docs/source/llms/llm-evaluate/index.rst

```diff
@@ -222,65 +209,209 @@ for LLM evaluation in MLflow. You can specify a custom list of metrics in the `e
 
 The full reference for supported evaluation metrics can be found `here <../../python_api/mlflow.html#mlflow.evaluate>`_.
 
-Metrics with LLM as the Judge
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create Custom heuristic-based LLM Evaluation Metrics
+****************************************************
+
+This is very similar to creating custom traditional metrics, with the exception of returning a :py:func:`mlflow.metrics.MetricValue` instance.
+Basically you need to:
+
+1. Implement a ``eval_fn`` to define your scoring logic, it must take in 2 args ``predictions`` and ``targets``.
+   ``eval_fn`` must return a :py:func:`mlflow.metrics.MetricValue` instance.
+2. Pass ``eval_fn`` and other arguments to ``mlflow.metrics.make_metric`` API to create the metric. 
+
+The following code creates a dummy per-row metric called ``"over_10_chars"``: if the model output is greater than 10, 
+the score is "yes" otherwise "no".
+
+.. code-block:: python
+
+    def eval_fn(predictions, targets):
+        scores = []
+        for i in range(len(predictions)):
+            if len(predictions[i]) > 10:
+                scores.append("yes")
+            else:
+                scores.append("no")
+        return MetricValue(
+            scores=scores,
+            aggregate_results=standard_aggregations(scores),
+        )
+
+
+    # Create an EvaluationMetric object.
+    passing_code_metric = make_metric(
+        eval_fn=eval_fn, greater_is_better=False, name="over_10_chars"
+    )
+
+To create a custom metric that is dependent on other metrics, include those other metrics' names as an argument after ``predictions`` and ``targets``. This can be the name of a builtin metric or another custom metric.
+Ensure that you do not accidentally have any circular dependencies in your metrics, or the evaluation will fail.
+
+The following code creates a dummy per-row metric called ``"toxic_or_over_10_chars"``: if the model output is greater than 10 or the toxicity score is greater than 0.5, the score is "yes" otherwise "no".
```

Comment:
```suggestion
The following code creates a dummy per-row metric called ``"toxic_or_over_10_chars"``; if the model output is greater than 10 or the toxicity score is greater than 0.5, the score is "yes", otherwise it is "no".
```